src/models/user.js

```diff
@@ -0,0 +1,18 @@
+export default Backbone.Model.extend({
+  urlRoot: 'http://tiny-lr.herokuapp.com/collections/users-bbs',
+
+  defaults: {
+    name: 'bbsoucy@gmail.com',
```

Comment:
You will not want to have a default email or password or you'll never be able to check for empty email submissions.